reference-architecture/osp-cli/ch3_generate_keypair.sh

```diff
@@ -0,0 +1,18 @@
+ssh-keygen -f ocp3_rsa -N "" -C "OpenShift KeyPair"
+Generating public/private rsa key pair.
```

Comment:
They are.  this is a bad hold over from the refarch doc sample, but not used in the scripts.
Removing.  *sigh*  Time to fine comb and not depend on you as my beta testers.